robolectric-shadows/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.robolectric</groupId>
     <artifactId>robolectric-parent</artifactId>
-    <version>3.2-SNAPSHOT</version>
+    <version>3.2-gradle-SNAPSHOT</version>
```

Comment:
It's just temporary, I'll drop this commit once snapshot publishing is working fully.